client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx

```diff
@@ -273,7 +275,6 @@ export const NumericTextInput = React.forwardRef<
           },
           ...sx,
         }}
-        inputMode="numeric"
```

Comment:
Could this be based on the `allowNegative` bool?
Feels like it might be an ok trade off? Or maybe that's just a weird ui thing?

Personally I'm ok with always seeing the whole keyboard, but would require switching to number mode to enter numbers which is a bit of a slow down...

Reply:
Sorry that's what you said you didn't like :( Need to read the PR properly first !

Reply:
Maybe yet another prop to control this so we can limit it just to R&R forms for now?